Dockerfile

```diff
@@ -1,23 +1,7 @@
-#base stage
-FROM archlinux:base-devel AS builder
+# builder image
+ARG  TAG=dev
+FROM hyperledger/iroha2-planner:$TAG AS builder
 
-ENV NIGHTLY=nightly-2022-08-15
-COPY ./rust-toolchain.toml .
-RUN set -eux && \
-    pacman -Syu rustup mold musl rust-musl --noconfirm && \
-    # toolchain: ./rust-toolchain.toml
-    rustup target add x86_64-unknown-linux-musl && \
-    rustup component add rust-src llvm-tools-preview  && \
-    # toolchain: $NIGHTLY
-    rustup install --profile default $NIGHTLY && \
-    rustup +$NIGHTLY target add x86_64-unknown-linux-musl wasm32-unknown-unknown && \
-    rustup +$NIGHTLY component add rust-src llvm-tools-preview && \
-    # cargo install
-    cargo install cargo-lints webassembly-test-runner && \
-    :
-
-# builder stage
-WORKDIR /iroha
 COPY . .
 RUN  rm -f rust-toolchain.toml
```

Comment:
Should we delete `rust-toolchain.toml` file before `cargo build` command or before `cargo chef prepare` && `cargo chef cook` commands on the `planner` stage?

Reply:
We should delete it from the repository. Add it to `.gitignore` too.

Reply:
Without `rust-toolchain.toml` we got this error while trying to compile ci-image: `error: rustup could not choose a version of rustup to run, because one wasn't specified explicitly, and no default is configured.`